addons/point_of_sale/static/tests/tours/helpers/TicketScreenTourMethods.js

```diff
@@ -226,7 +226,7 @@ export function receiptTotalIs(amount) {
 export function nthColumnContains(nRow, nCol, string){
     return [
         {
-            trigger: `.ticket-screen .order-row:nth-child(${nRow}) > .col:nth-child(${nCol}):contains("${string}")`,
+            trigger: `.ticket-screen .order-row:nth-last-child(${nRow}) > .col:nth-child(${nCol}):contains("${string}")`,
```

Comment:
I think you do not have to change this. It does not make much sense to start counting from the end and now that only test orders are shown, you can start counting from the start without problems